client/state/ui/notices/actions.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Internal dependencies
+ */
+import {
+	NEW_NOTICE,
+	REMOVE_NOTICE
+} from 'state/action-types';
+
+import { uniqueId } from 'lodash';
+
+export function createNoticeAction( status, text, options = {} ) {
+	return {
+		noticeId: uniqueId(),
+		duration: options.duration,
+		showDismiss: ( typeof options.showDismiss === 'boolean' ? options.showDismiss : true ),
+		type: NEW_NOTICE,
+		status: status,
+		text: text
+	};
+}
+
+export function removeNoticeAction( noticeId ) {
+	return {
+		noticeId: noticeId,
+		type: REMOVE_NOTICE
+	};
+}
+
+export function noticesMapDispatchToProps( dispatch ) {
```

Comment:
Ok, I remember now why I didn't use `bindActionCreators`.
I need `dispatch` in closure there to set callback for timeout for the notice to disappear. 
Every other solution does not give me a direct access to `dispatch`.
If I use `bindActionCreators`, I would have to create some ticker that keeps track of notices and `disappears` them. This ticker would have to be in controller (layout), because this is the only place that has direct access to `context.store.dispatch`

This can be a good solution, but definitely for a different PR.
For the time beeing, I can move this helper, but as I said - I have no better place.
`client/lib/redux-store-helpers` ?